test/fw/ptl/utils/plugins/ptl_test_info.py

```diff
@@ -128,26 +128,9 @@ def wantClass(self, cls):
         """
         Is the class wanted?
         """
-        if not issubclass(cls, PBSTestSuite):
```

Comment:
why did you remove this?

Reply:
This was there just ensure that we are not adding any class in self._tree which are not subclass of PBSTestSuite (as we want only test class to be added in self._tree) but now that is covered by loader plugin so we don't need to do it here. That's why it was removed